optuna/cli.py

```diff
@@ -861,7 +861,11 @@ def take_action(self, parsed_args: Namespace) -> int:
             study_name=parsed_args.study_name,
         )
 
-        state = TrialState[parsed_args.state.upper()]
+        if parsed_args.state is not None:
+            state: Optional[TrialState] = TrialState[parsed_args.state.upper()]
+        else:
+            state = None
```

Comment:
```suggestion
        if parsed_args.state is not None:
            state: Optional[TrialState] = TrialState[parsed_args.state.upper()]
```

Reply:
Sorry but please explain it in detail. 🙇
Would not `state` be unbounded if `parsed_args.state` None, and an error happen in Study.tell?

Reply:
Sorry you're right. Please ignore this comment. 🙇